### PR TITLE
Scheduling UI update

### PR DESCRIPTION
**Pull request by tomasol**
* Add link to crontab.guru for cron validation
* Linting
* Sort schedules consistently with workflowdef screen

### CODE REVIEW

**Review by ChmaraX — APPROVED** [review]

File: app/src/pages/workflowList/Scheduling/Scheduling.js [hunk 1]
```diff
@@ -40,24 +44,25 @@ const Scheduling = props => {
...
   }, []);
 
-
   const deselectActiveRow = () => {
     setActiveRow(null);
   };
 
-  const changeActiveRow = (i) => {
+  const changeActiveRow = i => {
+    console.log('changeActiveRow', i);
```
Comment (hotlib) [inline app/src/pages/workflowList/Scheduling/Scheduling.js]: console.log should ideally not be present

File: app/src/pages/workflowList/Scheduling/Scheduling.js [hunk 2]
```diff
@@ -40,24 +44,25 @@ const Scheduling = props => {
         deselectActiveRow();
       } else {
         console.log('err res', res, 'err', err);
-        const newError = (err != null) ? 'Network error: ' + err : 'Wrong response: ' + res;
+        const newError =
+          err != null ? 'Network error: ' + err : 'Wrong response: ' + res;
```
Comment (hotlib) [inline app/src/pages/workflowList/Scheduling/Scheduling.js:47]: using ES6 Template Strings makes it more readable

File: app/src/pages/workflowList/Scheduling/Scheduling.js [hunk 3]
```diff
@@ -40,24 +44,25 @@ const Scheduling = props => {
         deselectActiveRow();
       } else {
         console.log('err res', res, 'err', err);
```
Comment (hotlib) [inline app/src/pages/workflowList/Scheduling/Scheduling.js]: ideally console log should not be present

File: app/src/pages/workflowList/Scheduling/Scheduling.js [hunk 4]
```diff
@@ -72,16 +77,19 @@ const Scheduling = props => {
     setViewedPage(1);
   };
 
-  const deleteEntry = (schedulingEntry) => {
-    const path = global.backendApiUrlPrefix + "/schedule/" + schedulingEntry.name;
-    const req = superagent.delete(path).accept("application/json");
+  const deleteEntry = schedulingEntry => {
+    const path =
+      global.backendApiUrlPrefix + '/schedule/' + schedulingEntry.name;
```
Comment (hotlib) [inline app/src/pages/workflowList/Scheduling/Scheduling.js:80]: es6 Template Strings are better (imho)

File: app/src/pages/workflowList/Scheduling/Scheduling.js [hunk 5]
```diff
@@ -1,37 +1,41 @@
...
+import {GlobalContext} from '../../../common/GlobalContext';
+import {withRouter} from 'react-router-dom';
+
+const Scheduling = () => {
+  const global = useContext(GlobalContext);
   const [showSchedulingModal, setShowSchedulingModal] = useState(false);
   const [activeRow, setActiveRow] = useState();
   const [pagesCount, setPagesCount] = useState(1);
   const [data, setData] = useState(undefined);
+  // TODO: display error
```
Comment (hotlib) [inline app/src/pages/workflowList/Scheduling/Scheduling.js:26]: can this TODO be more elaborated? What should be implemented in the future?

File: app/src/pages/workflowList/Scheduling/Scheduling.js (hunk 3 above)
Comment (tomasol) [reply app/src/pages/workflowList/Scheduling/Scheduling.js]: outdated

File: app/src/pages/workflowList/Scheduling/Scheduling.js (hunk 1 above)
Comment (tomasol) [reply app/src/pages/workflowList/Scheduling/Scheduling.js]: outdated

File: app/src/pages/workflowList/Scheduling/Scheduling.js (hunk 2 above)
Comment (tomasol) [reply app/src/pages/workflowList/Scheduling/Scheduling.js:47]: done

File: app/src/pages/workflowList/Scheduling/Scheduling.js (hunk 4 above)
Comment (tomasol) [reply app/src/pages/workflowList/Scheduling/Scheduling.js:80]: I think this is readable

File: app/src/pages/workflowList/Scheduling/Scheduling.js (hunk 5 above)
Comment (tomasol) [reply app/src/pages/workflowList/Scheduling/Scheduling.js:26]: done

**Pull request merged**